### DISCUSSION TITLE
Problem MODBUS

### DISCUSSION THREAD

**Discussion by JonathanCoffeeAttitude** [artisan-roaster-scope/artisan#1050] [General]
Hi I need your help, I always used a computer with windows and I never had problems reading curves. But now I bought a Macbook air M2 2022, properly installed artisan but does not read the temperatures of hot air, beans or rate of raise, says 0. I downloaded the version ARTISAN-MAC-280 dmg, and I configured the toaster IMF. The USB port I use works and reads anything else, but it doesn’t work with artisan. I think it is an interface settings problem inherent in MODBUS settings ì, how can I solve the problem I’m desperate

**Comment by MAKOMO** [comment]
Did you install the serial driver for your USB2serial converter?

**Reply by JonathanCoffeeAttitude** [reply]
MMM I think no, because I don't know to do it, how can I to do? Thank you so much for your help

**Reply by MAKOMO** [reply]
IMF should now which serial driver you should install and how to do this.

**Comment by JonathanCoffeeAttitude** [comment]
So, can I to contact directly IMF for solution ? Are you sure ? Because in
artisan we have just IMF and IMF legacy. I hope that IMF cans to help me

Il giorno sab 17 dic 2022 alle 6:01 PM Marko Luther <
***@***.***> ha scritto:

**Reply by MAKOMO** [reply]
I don't know if you can contact them. You are their customer, right? Just for the notes, IMF never supported the Artisan project in any way.

**Comment by JonathanCoffeeAttitude** [comment]
All righ I will try contact them ! 😔😔😔 I’m a new roaster I don’t know

Il giorno sab 17 dic 2022 alle 7:08 PM Marko Luther <
***@***.***> ha scritto:

**Comment by JonathanCoffeeAttitude** [comment]
Just for to say, I choosed artisan because Paolo Scimone is my trainer, him
adviced me artisan.
I'm curious, what wrong if I have IMF ? I’m only a worker for roastery.
Have I any guilt for this ? I think no ! However I contacted IMF and I'll
wait for an answer. But you could helm me.

Il giorno sab 17 dic 2022 alle 7:09 PM Jonathan Bruno <
***@***.***> ha scritto:

**Reply by MAKOMO** [reply]
Nothing is wrong if you have an IMF. Just that we can not help you was we lack any information on IMF roasting machines as IMF is not supporting us, not financially but also not with any technical information. We just do not know anything about those machines. Sorry.
